<?xml version="1.0"?>
<launch>
    <param name="robot_description" textfile="$(find itr_rpc)/robots/2dofplanar.urdf" />
    <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" />
    <node name="kinematics_controller" pkg="itr_rpc" type="kinematics_controller.py" output="screen" />
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find itr_rpc)/config/basic.rviz" />
</launch>
